cpu/Makefile.include.gnu

```diff
@@ -9,6 +9,6 @@ endif
 export AS = $(PREFIX)as
 export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
-export OBJCOPY = $(PREFIX)objcopy
+export OBJCOPY = $(shell command -v $(PREFIX)objcopy gobjcopy objcopy touch | head -n 1)
```

Comment:
Why did you opt for `touch` as fallback? With the same arguments `objcopy` get, it will update the source's timestamp and create an empty destination file.

I would either use `cp` and hope for the best, or use `false` and fail once the command is used. What's better is a matter of taste I guess.